### DISCUSSION TITLE
Issue with Authorization with Azure DevOps pipeline

### DISCUSSION THREAD

**Discussion by rachaelph** [Azure/mlops-v2#17] [Q&A]
I have an issue when trying to run the bicep pipeline. 

ERROR: {'code': 'AuthorizationFailed', 'message': "The client '0733301f-3c42-4320-b5d0-f14a875f7106' with object id '0733301f-3c42-4320-b5d0-f14a875f7106' does not have authorization to perform action 'Microsoft.Resources/deployments/validate/action' over scope '/subscriptions/734a6f4e-ea8c-4bff-aaba-351502a8f700' or the scope is invalid. If access was recently granted, please refresh your credentials."}

**Comment by setuc** [comment]
Can you please crate an issue? Also can you share with us what steps did you use so that we can reproduce the issue.
